### ISSUE TITLE
Dropout during training

### ISSUE THREAD

**Issue by TobiasDijkhuis** [apax-hub/apax#499]
Including node dropout can improve generalization and reduce overfitting. 

See the Flax Linen [guide on dropout](https://flax-linen.readthedocs.io/en/latest/guides/training_techniques/dropout.html).

**Comment by M-R-Schaefer** [comment]
Worth considering, yes.
